scripts/provisioning/resetcmd.py

```diff
@@ -117,4 +125,36 @@ def DeleteFileOrDirWithRegex(self, path: str, regex: str):
           sys.stderr.write(f'ERROR: DeleteFileOrDirWithRegex(): Failed to delete: {file}, error: {str(e)}\n')
           raise e
 
-
+  def DeleteLdapAccountsUsers(self):
+    """Deletes all LDAP accounts and users."""
+    os.system('slapcat -n 3 -l conf_backup.ldif')
+    os.system('sed -i \'118,$ d\' conf_backup.ldif')
+    ldap_mdb_folder = "/var/lib/ldap"
+    for files in os.listdir(ldap_mdb_folder):
```

Comment:
Try and have a common function for mdb delete and use it in reset and cleanup. it will be in setupcmd.py